[2025-09-03T09:56:58.593+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-03T09:56:58.606+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: molecular_dynamics_analysis.run_small_analysis manual__2025-09-03T09:56:57.913723+00:00 [queued]>
[2025-09-03T09:56:58.608+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: molecular_dynamics_analysis.run_small_analysis manual__2025-09-03T09:56:57.913723+00:00 [queued]>
[2025-09-03T09:56:58.608+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-09-03T09:56:58.612+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): run_small_analysis> on 2025-09-03 09:56:57.913723+00:00
[2025-09-03T09:56:58.615+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=619) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-09-03T09:56:58.615+0000] {standard_task_runner.py:63} INFO - Started process 620 to run task
[2025-09-03T09:56:58.615+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'molecular_dynamics_analysis', 'run_small_analysis', 'manual__2025-09-03T09:56:57.913723+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/molecular_dynamics_dag.py', '--cfg-path', '/tmp/tmpb513v9qh']
[2025-09-03T09:56:58.617+0000] {standard_task_runner.py:91} INFO - Job 5: Subtask run_small_analysis
[2025-09-03T09:56:58.622+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-09-03T09:56:58.632+0000] {task_command.py:426} INFO - Running <TaskInstance: molecular_dynamics_analysis.run_small_analysis manual__2025-09-03T09:56:57.913723+00:00 [running]> on host ab9e9de509d0
[2025-09-03T09:56:58.660+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='molecular_dynamics_analysis' AIRFLOW_CTX_TASK_ID='run_small_analysis' AIRFLOW_CTX_EXECUTION_DATE='2025-09-03T09:56:57.913723+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-03T09:56:57.913723+00:00'
[2025-09-03T09:56:58.660+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-03T09:56:58.666+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-09-03T09:56:58.666+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/tmp/cpptraj/bin/cpptraj -p /opt/***/data/rerun_10us-traj/strip-wat-ions.caau-ol3-case-opc_NBfix-both-0BPhs_HMR_1.top -i /opt/***/scripts/nmr_unoe_ptraj.in']
[2025-09-03T09:56:58.669+0000] {subprocess.py:86} INFO - Output:
[2025-09-03T09:56:58.677+0000] {subprocess.py:93} INFO - Error: './caau_gHBfix21-tHBfix20_NBfix-both-0BPhs_rep1-10us_strip.traj': No such file or directory
[2025-09-03T09:56:58.678+0000] {subprocess.py:93} INFO - Error: Could not set up input trajectory './caau_gHBfix21-tHBfix20_NBfix-both-0BPhs_rep1-10us_strip.traj'.
[2025-09-03T09:56:58.678+0000] {subprocess.py:93} INFO - 	1 errors encountered reading input.
[2025-09-03T09:56:58.678+0000] {subprocess.py:93} INFO - Error: Error(s) occurred during execution.
[2025-09-03T09:56:58.678+0000] {subprocess.py:93} INFO - 
[2025-09-03T09:56:58.678+0000] {subprocess.py:93} INFO - CPPTRAJ: Trajectory Analysis. V6.29.14 (GitHub)
[2025-09-03T09:56:58.678+0000] {subprocess.py:93} INFO -     ___  ___  ___  ___
[2025-09-03T09:56:58.678+0000] {subprocess.py:93} INFO -      | \/ | \/ | \/ |
[2025-09-03T09:56:58.678+0000] {subprocess.py:93} INFO -     _|_/\_|_/\_|_/\_|_
[2025-09-03T09:56:58.678+0000] {subprocess.py:93} INFO - 
[2025-09-03T09:56:58.678+0000] {subprocess.py:93} INFO - | Date/time: 09/03/25 09:56:58
[2025-09-03T09:56:58.678+0000] {subprocess.py:93} INFO - | Available memory: 3.324 GB
[2025-09-03T09:56:58.679+0000] {subprocess.py:93} INFO - 
[2025-09-03T09:56:58.679+0000] {subprocess.py:93} INFO - 	Reading '/opt/***/data/rerun_10us-traj/strip-wat-ions.caau-ol3-case-opc_NBfix-both-0BPhs_HMR_1.top' as Amber Topology
[2025-09-03T09:56:58.679+0000] {subprocess.py:93} INFO - 	Radius Set: modified Bondi radii (mbondi)
[2025-09-03T09:56:58.679+0000] {subprocess.py:93} INFO - INPUT: Reading input from '/opt/***/scripts/nmr_unoe_ptraj.in'
[2025-09-03T09:56:58.679+0000] {subprocess.py:93} INFO -   [trajin ./caau_gHBfix21-tHBfix20_NBfix-both-0BPhs_rep1-10us_strip.traj]
[2025-09-03T09:56:58.679+0000] {subprocess.py:93} INFO - TIME: Total execution time: 0.0023 seconds.
[2025-09-03T09:56:58.679+0000] {subprocess.py:93} INFO - 
[2025-09-03T09:56:58.679+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2025-09-03T09:56:58.679+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-03T09:56:58.683+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/bash.py", line 243, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2025-09-03T09:56:58.685+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=molecular_dynamics_analysis, task_id=run_small_analysis, run_id=manual__2025-09-03T09:56:57.913723+00:00, execution_date=20250903T095657, start_date=20250903T095658, end_date=20250903T095658
[2025-09-03T09:56:58.689+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 5 for task run_small_analysis (Bash command failed. The command returned a non-zero exit code 1.; 620)
[2025-09-03T09:56:58.714+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-09-03T09:56:58.720+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-03T09:56:58.720+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
